src/containers/App/App.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { Route, HashRouter } from 'react-router-dom'
+import { Provider } from 'react-redux'
+import { hot } from 'react-hot-loader'
+import store from '../../redux/store'
+
+import Home from '../HomeContainer/HomeContainer'
+import About from '../../views/About/About'
+import KatathonData from '../KatathonContainer/KatathonContainer'
+
+import '../../styles/index.scss'
+
+const App = () => (
+  <Provider store={ store }>
+    <HashRouter>
+      <div>
```

Comment:
Yeas, I would prefer to use Switch here =)

Reply:
done and on the way